### PR TITLE
Playwright for firefox and webkit

### PR DESCRIPTION
**Pull request by TSKsmiley**

### CODE REVIEW

File: src/lib/components/project/ProjectItem.svelte [hunk 1]
```diff
@@ -78,12 +79,29 @@
 	button:focus-visible {
 		outline: 1px solid white;
 		outline-offset: -1px;
-	}
+	} */
```
Comment (DenFlyvendeGed) [inline src/lib/components/project/ProjectItem.svelte:82]: Delete if commented out

**Review by DenFlyvendeGed — COMMENTED** [review]
Works well after antivirus stops complaining ;)

**Review by DenFlyvendeGed — APPROVED** [review]
LGTM

**Review by atjn — APPROVED** [review]
Nice, LGTM

**Pull request merged**